docs/core/run-time-config/garbage-collector.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Garbage collector config settings
+description: Learn about run-time settings for configuring how the garbage collector manages memory.
+ms.date: 11/13/2019
+ms.topic: reference
+---
+# Run-time configuration options for garbage collection
+
+This page contains information about garbage collector (GC) settings that can be changed at run time. If you're trying to achieve peak performance of a running app, consider using these settings. However, most users will not need to concern themselves with these settings.
+
+On this page, settings are arranged into groups of settings that you can use in conjunction with each other to achieve a specific result.
+
+> [!NOTE]
+>
+> - These settings can also be changed dynamically by the app as it's running, so any run-time settings you set may be overridden.
+> - Some settings, such as [latency level](../../standard/garbage-collection/latency.md), are typically set only through the API at design time. Such settings are omitted from this page.
+
+## Flavors of garbage collection
+
+The two main flavors of garbage collection are workstation GC and server GC. For more information about differences between the two, see [Fundamentals of garbage collection](../../standard/garbage-collection/fundamentals.md#workstation-and-server-garbage-collection).
+
+The subflavors of garbage collection are background and non-concurrent.
+
+Use the settings described in this section to select flavors of garbage collection.
+
+### System.GC.Server/COMPlus_gcServer
+
+- Configures whether the application uses workstation garbage collection or server garbage collection.
+- Default: Workstation garbage collection (`false`).
+- For more information, see [Configure garbage collection](../../standard/garbage-collection/fundamentals.md#configuring-garbage-collection).
+
+| | Setting name | Values | Version introduced |
+| - | - | - | - |
+| **runtimeconfig.json** | `System.GC.Server` | `true` - server<br/>`false` - workstation | .NET Core 1.0 |
+| **Environment variable** | `COMPlus_gcServer` | 0 - workstation<br/>1 - server | .NET Core 1.0 |
+| **app.config for .NET Framework** | [GCServer](../../framework/configure-apps/file-schema/runtime/gcserver-element.md) | `true` - server<br/>`false` - workstation |  |
```

Comment:
if this is a .NET Core article, why are we listing the Framework ones? Just so people can map what they used to use? Otherwise, if this article is applicable to both, should we move it to under the standard folder?

Reply:
This is a tough one. It was requested that I add the app.config setting, and I think it's a good idea even though the article is Core-focused. Perhaps I can add a note at the top that says .NET Framework settings are added for reference purposes? I don't think we should move this under the standard folder though. I do refer to this article from the standard/gc/fundamentals.md article for info about configuration, but I guess we assume most people are configuring for Core?

Reply:
That's fine then if it's just for reference.